src/components/views/context_menus/MessageContextMenu.tsx

```diff
@@ -71,11 +71,11 @@ const ReplyInThreadButton = ({ mxEvent, closeMenu }: IReplyInThreadButton) => {
     if (Boolean(relationType) && relationType !== RelationType.Thread) return null;
 
     const onClick = (): void => {
-        if (!localStorage.getItem("mx_seen_feature_thread")) {
-            localStorage.setItem("mx_seen_feature_thread", "true");
+        if (!localStorage.getItem("mx_seen_feature_threadstable")) {
```

Comment:
It looks like you killed off the MessageActionBar bits of this, so doubt it'll work with only half the bits kept.